esphome/components/mpu6050/sensor/__init__.py

```diff
@@ -46,28 +42,26 @@
     state_class=STATE_CLASS_MEASUREMENT,
 )
 
-CONFIG_SCHEMA = (
-    cv.Schema(
-        {
-            cv.GenerateID(): cv.declare_id(MPU6050Component),
-            cv.Optional(CONF_ACCEL_X): accel_schema,
-            cv.Optional(CONF_ACCEL_Y): accel_schema,
-            cv.Optional(CONF_ACCEL_Z): accel_schema,
-            cv.Optional(CONF_GYRO_X): gyro_schema,
-            cv.Optional(CONF_GYRO_Y): gyro_schema,
-            cv.Optional(CONF_GYRO_Z): gyro_schema,
-            cv.Optional(CONF_TEMPERATURE): temperature_schema,
-        }
-    )
-    .extend(cv.polling_component_schema("60s"))
-    .extend(i2c.i2c_device_schema(0x68))
-)
+CONFIG_SCHEMA = cv.Schema(
+    {
+        cv.GenerateID(): cv.declare_id(MPU6050Sensor),
+        cv.Required(CONF_MPU6050_ID): cv.use_id(MPU6050Component),
```

Comment:
This will automatically get the id if there is only 1 parent in config
```suggestion
        cv.GenerateID(CONF_MPU6050_ID): cv.use_id(MPU6050Component),
```